lib/result.rb

```diff
@@ -0,0 +1,12 @@
+class Result
```

Comment:
The class is a helper class for `BurndownData`. So from my point of view it fits there quite well. If we want to use it as a stand-alone class we should find a better name. `Result` is too generic. I was thinking of `BurndownDataResult`, but then we can also just leave it inside the `BurndownData` class.